src/TraceEvent/TraceEvent.csproj

```diff
@@ -41,7 +40,7 @@
     <WarningLevel>3</WarningLevel>
     <DocumentationFile>bin\Debug\Microsoft.Diagnostics.Tracing.TraceEvent.xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>0649 0618</NoWarn>
+    <NoWarn>0649,0618</NoWarn>
```

Comment:
Note that this is a bug fix. It appears the project already intended to disable CS0618, but it wasn't applied because the syntax for `<NoWarn>` is a comma-separated list.